lib/rb/thrift.gemspec

```
@@ -27,7 +27,8 @@ Gem::Specification.new do |s|
 
   s.require_paths = %w[lib ext]
 
-  s.add_development_dependency 'bundler',            '~> 1.11'
+  s.add_development_dependency 'bundler',            '>= 1.0'
```

Ruby 2.7 uses Bundler v2 by default, so maybe we should do this?

```suggestion
  s.add_development_dependency 'bundler',            '>= 1.11'
```